app/styles/_layout.scss

```diff
@@ -49,6 +49,45 @@
     border: none;
     box-shadow: none;
   }
+}
+
+#fxa-settings-profile-header {
+  margin: 0 auto;
+
+  @include respond-to('big') {
+    width: 500px;
+  }
+
+  @include respond-to('small') {
+    width: 100%;
+  }
+}
+
+#fxa-settings-content {
+
+  background: $content-background-color;
+  margin: 0 auto;
+
+  @include respond-to('big') {
+    border-radius: $big-border-radius;
+    box-shadow: 0 1px 5px $input-row-border-color;
+    padding: 20px;  
+    width: 500px;
+  }
+
+  @include respond-to('small') {
+    border-bottom: 1px solid #b4bdc3;
+    border-top: 1px solid #b4bdc3;
```

Comment:
Add #b4bdc3 as constant in [/app/styles/_variables.scss](https://github.com/mozilla/fxa-content-server/blob/master/app/styles/_variables.scss)?